client/landing/gutenboarding/gutenboard.tsx

```diff
@@ -105,7 +105,7 @@ export function Gutenboard() {
 							value={ [ onboardingBlock.current ] }
 							settings={ { templateLock: 'all' } }
 						>
-							<div className="edit-post-layout__content">
+							<div className="gutenboard__edit-post-layout-content edit-post-layout__content ">
```

Comment:
I'd like to do a scrub of these some time soon, to make things more `edit-post` agnostic.

Reply:
I think `edit-post-layout__content` has been removed from Gutenberg as well. (in https://github.com/WordPress/gutenberg/pull/18044). We had to update our styles in https://github.com/Automattic/wp-calypso/pull/37863